docs/topic/troubleshooting.md

```diff
@@ -0,0 +1,28 @@
+# Troubleshooting and debugging
+
+These sections describe a few common and useful tips for troubleshooting our infrastructure.
+
+## Roll back (revert) a helm deploy
+
+Sometimes it is useful to simply **revert** a Kubernetes deployment with Helm.
+For example, if you've manually deployed something via `helm install`, and notice that something is wrong with our deployment.
+
+:::{note}
+Ideally, this would have happened automatically via CI/CD, but sometimes a manual deploy is still necessary!
+:::
+
+If you'd simply like to revert back to the state of the Kubernetes infrastructure from before you ran `helm install`, try the following commands:
+
+- **Get the deployment name and revision number for the latest deploy**. To do so, run this command:
+  ```bash
+  helm list -n {{NAMESPACE}}
+  ```
+- Roll back the deployment to the previous revision, using the information output from the above command:
+  
+  ```bash
+  helm rollback -n {{NAMESPACE}} {{DEPLOYMENT_NAME}} {{REV_NUM}}-1
+  ```
+
+  The {{REV_NUM - 1}} simply means "deploy the previous revision number"
```

Comment:
```suggestion
  The {{REV_NUM - 1}} simply means "deploy the previous revision number".
  Usually, `NAMESPACE` and `DEPLOYMENT_NAME` are identical, but always best to double check.
```

Reply:
> Usually, `NAMESPACE` and `DEPLOYMENT_NAME` are identical, but always best to double check.

Maybe adding a few lines about how to double-check?

Reply:
It should be obvious from the output of `helm list -n NAMESPACE` above, no? e.g.

```
$ helm list -n staging
NAME   	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART        	APP VERSION
staging	staging  	57      	2021-09-23 12:39:49.844059 +0100 BST	deployed	daskhub-0.1.0	1.0
```

Reply:
Yep, I was reading this comment isolated from the above lines, my bad 😉 .